typify-impl/src/type_entry.rs

```diff
@@ -1635,7 +1643,10 @@ impl TypeEntry {
                 } else {
                     let key_ident = key_ty.type_ident(type_space, type_mod);
```

Comment:
github it's letting me comment, but at line 1642 we emit a `::serde_json::Map<::std::string::String, ::serde_json::Value>`. I think this is the right thing to do in this case, but I do want to point out that this is an exceptional case. What do you think about documenting it?

Reply:
I believe this is adequately documented [here](https://github.com/oxidecomputer/typify/blob/main/README.md?plain=1#L49-L51) in the readme already.

Happy to add further clarification if you'd like.

Reply:
I meant: document that the override doesn't apply to this.